File: Assets/VirtualCity/ProtoDefine/Java/system/RspUpdateUserInfoMessage.java
package com.kingston.jforgame.server.game.system.message;

import com.baidu.bjf.remoting.protobuf.FieldType;
import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import com.kingston.jforgame.server.game.Modules;
import com.kingston.jforgame.server.game.system.SystemDataPool;
import com.kingston.jforgame.server.game.system.entity.UserInfoMap;
import com.kingston.jforgame.socket.annotation.MessageMeta;
import com.kingston.jforgame.socket.message.Message;

import java.util.List;

@MessageMeta(module = Modules.SYSTEM,cmd = SystemDataPool.RSP_UPDATEUSERINFO)
public class RspUpdateUserInfoMessage extends Message {
    @Protobuf(order = 1)
    private int code;
    @Protobuf(order = 2)
    private String tips;
    @Protobuf(order = 3)
    private Long accountId;



    @Protobuf(order = 4,fieldType = FieldType.OBJECT)
    private List<UserInfoMap> userInfoMap;

    public RspUpdateUserInfoMessage() {
    }

    public RspUpdateUserInfoMessage(int code, String tips, Long accountId, List<UserInfoMap> userInfoMap) {
        this.code = code;
        this.tips = tips;
        this.accountId = accountId;
        this.userInfoMap = userInfoMap;
    }

    public List<UserInfoMap> getUserInfoMap() {
        return userInfoMap;
    }

    public void setUserInfoMap(List<UserInfoMap> userInfoMap) {
        this.userInfoMap = userInfoMap;
    }

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public String getTips() {
        return tips;
    }

    public void setTips(String tips) {
        this.tips = tips;
    }

    public Long getAccountId() {
        return accountId;
    }

    public void setAccountId(Long accountId) {
        this.accountId = accountId;
    }
}

File: Assets/VirtualCity/ProtoDefine/RspUpdateUserInfoMessage.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]



public class RspUpdateUserInfoMessage {
[ProtoMember(1)]
    public int code;
[ProtoMember(2)]
    public string tips;
[ProtoMember(3)]
    public long? accountId;



[ProtoMember(4)]
    public List<UserInfoMap> userInfoMap;

    public RspUpdateUserInfoMessage() {
    }

    public RspUpdateUserInfoMessage(int code, string tips, long? accountId, List<UserInfoMap> userInfoMap) {
        this.code = code;
        this.tips = tips;
        this.accountId = accountId;
        this.userInfoMap = userInfoMap;
    }

    public List<UserInfoMap> getUserInfoMap() {
        return userInfoMap;
    }

    public void setUserInfoMap(List<UserInfoMap> userInfoMap) {
        this.userInfoMap = userInfoMap;
    }

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public string getTips() {
        return tips;
    }

    public void setTips(string tips) {
        this.tips = tips;
    }

    public long? getAccountId() {
        return accountId;
    }

    public void setAccountId(long? accountId) {
        this.accountId = accountId;
    }
}
}